concordium-consensus/stack.static.yaml

```diff
@@ -1,5 +1,19 @@
 resolver: lts-24.0
 
+# To be able to produce a statically linked shared library, we need to use a ghc
+# and libraries variant compiled with Position Independent Code (PIC) enabled.
+# Such a build is not provided by default, so instead we use a custom build.
+#
+# See `/scripts/build-static-ghc.sh` for how to do this.
+ghc-variant: static-build
+setup-info:
+  ghc:
+    linux64-custom-static-build-tinfo6:
+      9.10.2:
+        url: "/home/lime/Concordium/concordium-node/ghc2/ghc-9.10.2/_build/bindist/ghc-9.10.2-x86_64-unknown-linux.tar.xz"
```

Comment:
When the build-static-ghc workflow is working, this needs to point to the resource